### PR TITLE
Adds some redis

### LINKED ISSUE #10: Externalize session data to make the app more stateless/production-ready
Similar to [this comment about `cf-ex-drupal`](https://github.com/18F/cf-ex-drupal/issues/5), we should address this limitation (from the README) with instructions for using the RDS or Redis services, instead of leaving it as a caution:

> This is not an issue with Wordpress specifically, but PHP stores session information to the local disk. As mentioned previously, the local disk for an application on CloudFoundry is ephemeral, so it is possible for you to lose session and session data. If you need reliable session storage, look at storing session data in an SQL database or with a NoSQL service.

### PR DESCRIPTION
**Pull request by gboone**
This seems to work right out the box. Closes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,8 +257,7 @@ Consider using [continuous integration](https://cloud.gov/docs/apps/continuous-d
 ### Recommendations
 
 1. You will probably want to connect your app to some kind of SMTP service to send transactional emails like password resets.
-1. The S3 Uploads plugin rewrites the URLs used by WordPress but does not flush the rewrite rules table automatically. To get around this, you can [run a task](https://cloud.gov/docs/getting-started/one-off-tasks/) to flush the rewrite rules after every `cf push` of your app. You can also automate those tasks by using [continuous integration](https://cloud.gov/docs/apps/continuous-deployment/).
-
+1. If you're running this application in production, you probably want to [connect a `redis32` service](https://cloud.gov/docs/services/redis/) to handle local session storage. Like most PHP applications, Wordpress stores session data (like permalinks and any other [transients](https://codex.wordpress.org/Transients_API) your app might use) to the local file system. As with uploads, these will get cleared periodically due to the ephemeral file system.
```
Comment (jmcarp) [inline README.md]: Let's add a `cf create-service` command for the redis service instance and add it the example `manifest.yml`. Based on https://docs.cloudfoundry.org/buildpacks/php/gsg-php-sessions.html, it looks like the service name is significant unless we also set `REDIS_SESSION_STORE_SERVICE_NAME`, so it's important to either give guidance on the name or set the environment variable.

**Comment by jmcarp** [comment]
If I wanted to test this, how would I verify that redis sessions are working? Is it enough to run multiple instances and verify that authentication works?

**Comment by gboone** [comment]
I think the hypothesis is:
1. Setup multiple instances
1. Log in
1. Restage (so the instances switch)
1. See if you're still logged in

Does that sound right?

**Comment by mogul** [comment]
Yes, that would work. You can also try multiple instances *without* this change and verify you see the problem without the fix. You might need to reload a few times to see if it's working/not working.

**Comment by mogul** [comment]
Made a bunch of changes and it seems to work... Was able to restage without being logged out.

**Comment by mogul** [comment]
@pburkholder want to review and merge?

**Comment by mogul** [comment]
It looks like Wordpress stores all client session data in cookies, and doesn't rely on server-side session storage at all! So this was all a snipe-hunt. Going to open a new PR with the small fixes I made that are worthwhile.

**Comment by gboone** [comment]
Hmmm, this means the issue I was seeing with permalinks wasn't related to session storage, then. _Bummer_

**Pull request closed without merging**